tests/src/core/parse.js

```diff
@@ -1,11 +1,14 @@
 import * as fs from 'fs'
 import { expect } from 'chai'
+import sinon from 'sinon'
 import parse from 'eslint-module-utils/parse'
 
 import { getFilename } from '../utils'
 
 describe('parse(content, { settings, ecmaFeatures })', function () {
   const path = getFilename('jsx.js')
+  const parseStubParser = require('./parseStubParser')
+  const parseStubParserPath = require.resolve('./parseStubParser')
```

Comment:
@ljharb This `require` was left here like that on purpose, this is not just a dependency like any other, this is a stub which must be in a file that can be literally `require`d because of the way `eslint`, and following it, `eslint-import-resolver`, resolves the parser function via `require` function (and we can do nothing about that).